gradle.properties

```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=2.1.2
+VERSION_NAME=2.1.3
 VERSION_CODE=9
```

Comment:
I think you also need to bump the `VERSION_CODE` to `10`; it might not actually matter but I vaguely recall that it might be used by some portion of the Android build logic.